### ISSUE TITLE
cannot import name 'url_decode' from 'werkzeug.urls' (no duplicates!!!)

### ISSUE THREAD

**Issue by AnubisXHyperX** [maxcountryman/flask-login#782]
After upgrading to Flask 3.0.0, the following imports no longer work in flask_login/utils.py:

```
from werkzeug.urls import url_decode
from werkzeug.urls import url_encode
```
To Reproduce

In a project that uses Flask 3.0.0 and Flask Login 0.6.2, include from flask_login import LoginManager and initialize LoginManager.
Run tests or the project
Observe ImportError: cannot import name 'url_decode' from 'werkzeug.urls' 

*when checking for duplicates and even starting from scratch still happens (even if I try to install requirement on a new docker container)*

Expected behavior
No errors when using LoginManager.

Desktop (please complete the following information):

OS: macOS Sonoma
Version: 14.0

**Comment by ThiefMaster** [comment]
#778 apparently fixed this (not on pypi yet though), even though the diff view doesn't properly show it...

**Comment by niralkk** [comment]
> #778 apparently fixed this (not on pypi yet though), even though the diff view doesn't properly show it...

Hey @ThiefMaster do you happen to know how soon it will be Pypi ? How long is the deployment process ?

**Comment by ThiefMaster** [comment]
no idea, i'm not a maintainer of this package

**Comment by davidism** [comment]
Please search for duplicates first.

**Comment by davidism** [comment]
Duplicate of #744, fixed by #746. A volunteer maintainer will work on making a release when they have time available.
